guides/tech-team-promotion-process.md

```diff
@@ -0,0 +1,360 @@
+[the templates](https://drive.google.com/drive/folders/1q4idxObiYhqOAWuzudFQcGAk_70yZk-I)
+explaining why you feel you are ready for a promotion. This pack includes a
+short personal statement on why you feel you should receive a promotion, a
+statement from your line manager on the same topic, and evidence to demonstrate
+why you would meet the needs of the role you would be promoted into. Good
+evidence for this might include:
+
+- evidence of you doing the things described with how they went
+- feedback from people you have worked with or clients that demonstrate your
+  ability to do the things described
+- evidence of similar activities that give you and your line manager confidence
+  you are ready to take on the responsibilities of the role
+
+Make sure you address all of the needs of the level and don’t leave any gaps.
+It’s ok to acknowledge that you don’t have direct experience of something in the
+framework. It’s better to do that than have the panel spend time working out if
+you do or not and run out of time to properly consider your strengths.
+
+You should involve your line manager in the process, but they are not allowed to
+block your promotion. They should support you with developing your promotion
+pack, but you can put yourself forwards for promotion without your line
+manager’s support.
+
+If you are reading this thinking about a future promotion, you might want to
+consider keeping a [brag document](https://jvns.ca/blog/brag-documents/) or
+using the Wins feature of Progression to keep track of evidence you might want
+to use in the future.
+
+### Submitting the application
+
+Email a link to your promotion pack to a member of the HR team, your Head of
+Discipline and your line manager. You need to also transfer ownership of any
+documents to a member of the HR team so it can go into your file and make sure
+your Head of Discipline has access to modify sharing settings so they can share
+it with panelists.
+
+While optional, it’s recommended that you share the pack with your Head of
+Discipline at least a week in advance of the application deadline so they can do
+a pre-check and make suggestions based on their experience reviewing other
+promotions, such as places where you could provide more detail.
+
+### Promotion panel
+
+While you create your promotion pack, your Head of Discipline decides on an
+appropriate panel to review your application.
+
+A panel is made up of:
+
+- someone from your discipline at least as senior as the role you are applying
+  for, if possible
+- your Head of Discipline (unless they are your line manager)
+- another member of the technology leadership team (Head of Development, Head of
+  Technical Operations, and Technical Architects)
+- two Heads of Discipline, Principals, or Leads from outside the technology team
+
+The panel must not include your line manager, buddy, or anyone in a close
+personal relationship with you to avoid conflicts of interest. This includes
```

Comment:
I have a passing comment on this point that shouldn't this hold up but could be useful to discuss.

I understand why conflict of interest is important to avoid for the panel. As someone who has been at the company for a long long I've worked with lots of people, quite a lot. The line between friend and colleague is difficult to be specific about so the right thing to do is to abstain if in doubt. There may be lots of people in the technical team for whom this applied for someone in my position.

Whilst I'm happy that abstaining from the panel is right I wonder if there's something else we could do in this situation to not waste all the professional context. Perhaps being invited to write a testimonial that could be considered as part of the promotional pack?

Reply:
I love that idea. I think testimonials from colleagues as part of the pack are a nice idea. Do you see that as something the applicant asks for and includes themselves, or something gathered by the Head of after submission?

Reply:
Once the panel was confirmed and that information shared to the applicant, they could use that information to ask any(?) number of their colleagues to write one for their pack. I'm not sure if the timeline supports that but I think we would want to let the person invited to the panel abstain first if _they_ felt conflicted. 

If we invite the applicant to ask for any number of testimonials ahead of time, asking for a panelist to write one would perhaps confuse things.